Decode/PerfItemValue.cs

```diff
@@ -1529,8 +1641,8 @@ public string GetString()
             const string Separator = ", ";
             int count;
             Text.Encoding? encodingFromBom;
-            var baseEncoding = this.Type.EncodingAndArrayFlags.BaseEncoding();
-            switch (this.Type.EncodingAndArrayFlags.BaseEncoding())
+            var baseEncoding = this.Metadata.Encoding;
```

Comment:
Note: the `EncodingAndArrayFlags` property no longer exists. But the `Encoding` property is the same as `EncodingAndArrayFlags.BaseEncoding()`.